### ISSUE TITLE
Argument for testset instead of trainingset?

### ISSUE THREAD

**Issue by htdbui** [julianfaraway/LMP#8]
https://github.com/julianfaraway/LMP/blob/6b6f59e1496a43bf80ae8675c9e22fc000229bd1/pyscripts/varsel.py#L216

Should it be testpred[0] = rmse(testfat.brozek, np.mean(testfat.brozek)) instead of rmse(testfat.brozek, np.mean(trainfat.brozek)) ?

**Comment by julianfaraway** [comment]
Yes you are correct. Fortunately it makes very little numerical difference.
